mojo/mojo_library.bzl

```diff
@@ -56,6 +57,14 @@ def _mojo_library_implementation(ctx):
 mojo_library = rule(
     implementation = _mojo_library_implementation,
     attrs = {
+        "copts": attr.string_list(
+            doc = """\
+Additional compiler options to pass to the Mojo compiler.
+
+NOTE: copts from --mojocopt and mojo_toolchain.copts are not passed to 'mojo
```

Comment:
is `mojocopt` a typo?

Reply:
bazel uses `copt` as the suffix even if the compiler doesn't end in `c` if that's what you mean? `mojo compiler opt` is how i read it